gulpfile.js

```diff
@@ -124,7 +124,7 @@ gulp.task('apps', appsBuild);
 const debugAppsBuild = gulp.series(gulp.parallel(clean_debug, gulp.series(clean_dist, debugDistBuild)), debug, gulp.series(cordova_apps(false)), gulp.parallel(listPostBuildTasks(DEBUG_DIR)));
 
 const debugBuildNoStart = gulp.series(debugDistBuild, debug, gulp.parallel(listPostBuildTasks(DEBUG_DIR)));
-const debugBuild = gulp.series(debugBuildNoStart, start_debug);
+const debugBuild = gulp.series(clean_dist, debugBuildNoStart, start_debug);
```

Comment:
The clean_dist was removed from the debug build to accelerate the build while testing. There isn't a better fix?

Reply:
This step takes only a 1 or 2 seconds on my system.

Reply:
i don't mind cleaning as well. my alias command does this manually, nice to return, imho.

Reply:
We can do any optimization after we fix this bug.

Reply:
The problem is not the clean_dist command, at least before the change, the rest of the debugBuild was a lot faster if the files were there and a lot slower if it must start from scratch. I don't know if this applies to the current version, the build has changed a lot since then.

Reply:
More info here: https://github.com/betaflight/betaflight-configurator/pull/1144

Reply:
`yarn && yarn start` the second times takes about 10 seconds here.

Reply:
that's fair too. i;ve not seen problems, since i always force a clean.  this line can be reverted if desired.

Reply:
Then we get the repeating dist dist dist issue